### DISCUSSION TITLE
RaspberryPi client and server versions

### DISCUSSION THREAD

**Discussion by patmu** [BearWare/TeamTalk5#2001] [Q&A]
### Description

Hi

Currently I'm using TeamTalk 5.8 on RaspberryPi Buster.

I would like to update to TeamTalk 5.14, but there is no release for RasperryPi anymore. Not for Buster and not for Bullseye.

Are there plans to release versions for the RaspberryPi plattform again? It would be very helpfull to support the RaspberryPi plattform in the future. Pi's are a very good plattform to build small intercom stations and run the server on it as well.

Thank you for your considerations.

Best regards,
Patrick

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by bear101** [comment]
Raspbian Buster comes with Qt 5.5 but the TeamTalk client requires Qt 5.15. Therefore the TeamTalk client is no longer available. You can download TeamTalk v5.13 for Raspbian Buster in the download section.

The TeamTalk Pro Server is available for Raspbian Buster.
